worker/games.py

```diff
@@ -840,7 +840,10 @@ def run_games(worker_info, password, remote, run, task_id):
     repo_url = run["args"].get("tests_repo", REPO_URL)
     games_concurrency = int(worker_info["concurrency"]) // threads
 
-    start_game_index = task_id * task["num_games"] + input_total_games
+    opening_offset=task.get("start",task_id * task["num_games"])
```

Comment:
space before comma,

Reply:
Ok. I think black fixed it. Contained in latest push.

Reply:
Thank you